### ISSUE TITLE
404 gitmojis.json

### ISSUE THREAD

**Issue by tianyingchun** [arvinxx/gitmoji-commit-workflow#657]
https://github.com/carloscuesta/gitmoji/blob/master/src/data/gitmojis.json

this repo has been removed.

**Comment by caiotracera** [comment]
Hey, @tianyingchun!
For now, you can take the content from [this link](https://github.com/carloscuesta/gitmoji/blob/master/packages/gitmojis/src/gitmojis.json) and paste on `node_modules/commitlint-plugin-gitmoji/lib/gitmojis.json` or something like that.

**Comment by tianyingchun** [comment]
can these be installed automatically? always need mananully?

**Comment by caiotracera** [comment]
Yes, the process can be automatic again, but I think we need to update the [`packages/commitlint-plugin`](https://github.com/arvinxx/gitmoji-commit-workflow/blob/master/packages/commitlint-plugin/package.json#L36) to use at least the version 3.13.1, that seems to be root cause of this problem: https://github.com/carloscuesta/gitmoji/releases/tag/v3.13.1

I don't know if @arvinxx is working on that, so can you open a PR with this change? I am afraid that I won't have time to work on that anytime soon :(

**Comment by caiotracera** [comment]
Maybe this fix can be included on #656

**Comment by tianyingchun** [comment]
:) waiting this release, i am really not familar with implementation of gitmoji

**Comment by patevs** [comment]
As a workaround, you can set the `GITMOJI_URL` environment variable to the working URL.

E.g., Using PowerShell, run the following command:
```
$ENV:GITMOJI_URL="https://raw.githubusercontent.com/carloscuesta/gitmoji/master/packages/gitmojis/src/gitmojis.json"
```

**Comment by arvinxx** [comment]
Fix in [2.2.8](https://github.com/arvinxx/gitmoji-commit-workflow/releases/tag/commitlint-config-gitmoji%402.2.8)
